developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -201,7 +201,15 @@ Supported properties
 | <nc:is-encrypted />           | Whether the folder is end-to-end encrypted.     | | ``0`` for ``false``                                                                |
 |                               |                                                 | | ``1`` for ``true``                                                                 |
 +-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------------------+
-| <oc:tags />                   | List of user specified tags.                    | ``<oc:tag>test</oc:tag>``                                                            |
```

Comment:
Tags still exist, that should be the favorite tag e.g.

Reply:
TIL